linux_os/guide/services/ftp/disabling_vsftpd/package_vsftpd_removed/rule.yml

```diff
@@ -29,6 +30,10 @@ references:
     iso27001-2013: A.12.1.2,A.12.5.1,A.12.6.2,A.14.2.2,A.14.2.3,A.14.2.4,A.9.1.2
     cis-csc: 11,14,3,9
     stigid@rhel8: RHEL-08-040360
+    nist@sle15: IA-5(1)(c),IA-5(1).1(v),CM-7,CM-7.1(ii),CM-7 a
```

Comment:
CM-7 a should be CM-7(a)

Reply:
Thanks  - should be ok in f9febde30d3516c821d74f9054e855f7ea5c185c